# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-08-14 21:30+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: vk_bot/service/handlers.py:14 vk_bot/service/handlers.py:32
#: vk_bot/service/handlers.py:35
msgid "cmd.owe"
msgstr "должен"

#: vk_bot/service/handlers.py:14 vk_bot/service/handlers.py:32
#: vk_bot/service/handlers.py:35
msgid "monthly"
msgstr "ежемесячно"

#: vk_bot/service/handlers.py:49 vk_bot/service/handlers.py:60
#: vk_bot/service/handlers.py:63
msgid "cmd.pay"
msgstr "заплатить"

#: vk_bot/model/debt.py:27
msgid "debt.info.monthly"
msgstr ""
"Название: {}\n"
"Дата: {}\n"
"Период: {}\n"
"Сумма: {}\n"
"Одолжил: [id{}|{}]"

#: vk_bot/model/debt.py:28
msgid "debt.info.once"
msgstr ""
"Название: {}\n"
"Дата: {}\n"
"Сумма: {}\n"
"Одолжил: [id{}|{}]"

#: vk_bot/service/handlers.py:28
msgid "January"
msgstr "Январь"

#: vk_bot/service/handlers.py:28
msgid "February"
msgstr "Февраль"

#: vk_bot/service/handlers.py:28
msgid "March"
msgstr "Март"

#: vk_bot/service/handlers.py:28
msgid "April"
msgstr "Апрель"

#: vk_bot/service/handlers.py:28
msgid "May"
msgstr "Май"

#: vk_bot/service/handlers.py:28
msgid "June"
msgstr "Июнь"

#: vk_bot/service/handlers.py:28
msgid "July"
msgstr "Июль"

#: vk_bot/service/handlers.py:28
msgid "August"
msgstr "Август"

#: vk_bot/service/handlers.py:28
msgid "September"
msgstr "Сентябрь"

#: vk_bot/service/handlers.py:28
msgid "October"
msgstr "Октябрь"

#: vk_bot/service/handlers.py:28
msgid "November"
msgstr "Ноябрь"

#: vk_bot/service/handlers.py:28
msgid "December"
msgstr "Декабрь"

#: vk_bot/service/commands.py:107 vk_bot/service/commands.py:111
#: vk_bot/service/commands.py:112 vk_bot/service/commands.py:155
#: vk_bot/service/commands.py:167 vk_bot/service/commands.py:174
#: vk_bot/service/commands.py:173 vk_bot/service/commands.py:184
#: vk_bot/service/commands.py:200 vk_bot/service/commands.py:215
msgid "debts.info"
msgstr ""
"Введите номер долга, затем через двоеточие сумму, которую хотите заплатить в "
"пользу данного долга. Перечисляйте долги через пробел. Пример: 1:250 2:200. "
"Для отмены текущей команды отправьте -1 {}{}"

#: vk_bot/service/commands.py:97 vk_bot/service/commands.py:98
#: vk_bot/service/commands.py:141 vk_bot/service/commands.py:153
#: vk_bot/service/commands.py:160 vk_bot/service/commands.py:159
#: vk_bot/service/commands.py:169 vk_bot/service/commands.py:185
#: vk_bot/service/commands.py:200
msgid "exception.no_debts"
msgstr "Долгов не найдено"

#: vk_bot/service/commands.py:16 vk_bot/service/commands.py:17
#: vk_bot/service/commands.py:19
msgid "exception.amount"
msgstr "Сумма долга должна быть больше или равна 1"

#: vk_bot/service/commands.py:19 vk_bot/service/commands.py:20
#: vk_bot/service/commands.py:22
msgid "exception.owe_himself"
msgstr "Пользователь не может задолжать самому себе"

#: vk_bot/service/commands.py:28 vk_bot/service/commands.py:29
#: vk_bot/service/commands.py:31
msgid "owe.period"
msgstr ""
"В текущий или следующий месяц внести первый платеж? Для текущего периода "
"отправте цифру 0, для следующего - цифру 1. Для отмены текущей команды "
"отправьте -1"

#: vk_bot/service/commands.py:33 vk_bot/service/commands.py:34
#: vk_bot/service/commands.py:49 vk_bot/service/commands.py:51
#: vk_bot/service/commands.py:54
msgid "owe.debt.register"
msgstr ""
"Информация о долге была зарегестрирована. После того, как все должники "
"подтвердят данный долг, информация будет полностью сохранена в системе"

#: vk_bot/service/commands.py:48 vk_bot/service/commands.py:88
#: vk_bot/service/commands.py:110 vk_bot/service/commands.py:117
#: vk_bot/service/commands.py:116 vk_bot/service/commands.py:126
#: vk_bot/service/commands.py:142 vk_bot/service/commands.py:153
msgid "owe.debt.saved"
msgstr "Информация о долге успешно сохранена"

#: vk_bot/routes.py:42 vk_bot/routes.py:55
msgid "exception.cmd.not_recognised"
msgstr ""
"Команда не распознана, проверьте формат. Для просмотра доступных команд "
"напишите \"помощь\""

#: vk_bot/service/commands.py:49 vk_bot/service/commands.py:46
#: vk_bot/service/commands.py:48 vk_bot/service/commands.py:51
msgid "owe.debt.confirm"
msgstr ""
"#{}\n"
"Название: {}\n"
"Одолжил: [id{}|{}]\n"
"{}\n"
"Для подтверждения данного долга отправьте 1 в ОТВЕТЕ на это сообщение. Также "
"можете выбрать несколько долгов и переслать их сюда боту, отправив 1 в "
"сообщении. На подтверждение даётся 24 часа, иначе информация о долге будет "
"удалена"

#: vk_bot/service/commands.py:96 vk_bot/service/commands.py:103
#: vk_bot/service/commands.py:102 vk_bot/service/commands.py:112
#: vk_bot/service/commands.py:114 vk_bot/service/commands.py:117
msgid "confirm.confirmed"
msgstr "Выбранные долги и платежи подтверждены"

#: vk_bot/service/handlers.py:87 vk_bot/service/handlers.py:89
msgid "exception.confirm.bad_id"
msgstr ""
"Произошла ошибка при подтверждении, проверьте отправителя пересылаемого "
"сообщения"

#: vk_bot/service/handlers.py:90 vk_bot/service/handlers.py:92
msgid "exception.confirm.not_confirmed"
msgstr "Долг не подтвержден, отправьте в ответе цифру 1 для подтверждения"

#: vk_bot/service/commands.py:82 vk_bot/service/commands.py:91
#: vk_bot/service/commands.py:81 vk_bot/service/commands.py:96
#: vk_bot/service/commands.py:84 vk_bot/service/commands.py:119
#: vk_bot/service/commands.py:87 vk_bot/service/commands.py:122
msgid "exception.confirm.outdated"
msgstr "Информация о долге не найдена"

#: vk_bot/service/commands.py:85 vk_bot/service/commands.py:87
#: vk_bot/service/commands.py:89 vk_bot/service/commands.py:92
msgid "exception.confirm.user_not_found"
msgstr "Данный пользователь не должен подтверждать данный долг"

#: vk_bot/routes.py:40
msgid "exception.confirm.bad_forward"
msgstr "Одно из пересылаемых сообщений не соответствует формату"

#: vk_bot/service/temp.py:23 vk_bot/service/temp.py:27
msgid "exception.invalid_option"
msgstr "Выбрана несуществующая опция"

#: vk_bot/service/temp.py:55 vk_bot/service/temp.py:81
#: vk_bot/service/temp.py:84
msgid "exception.invalid_format"
msgstr "Формат опции некорректен"

#: vk_bot/service/commands.py:192 vk_bot/service/commands.py:214
#: vk_bot/service/commands.py:229
msgid "cmd.pay_accept"
msgstr ""
"#{}\n"
"Название: {}\n"
"Сумма: {}\n"
"{}\n"
"Подтвердите, что пользователь [id{}|{}] заплатил в пользу данного долга "
"указанную сумму, отправив 1 в ОТВЕТЕ. Также можете выбрать несколько "
"подтверждений и переслать их боту в личном сообщении, отправив 1 в "
"сообщении. На подтверждение даётся 24 часа, иначе информация будет удалена"

#: vk_bot/service/commands.py:222 vk_bot/service/commands.py:238
msgid "cmd.pay_register"
msgstr ""
"Сумма к оплате - {}. После того, как оплата будет подтверждена, информация "
"полностью будет сохранена в системе"

#: vk_bot/service/temp.py:68 vk_bot/service/temp.py:58
msgid "cmd.cancel"
msgstr "Предыдущая команда отменена"

#: vk_bot/service/util.py:17
msgid "exception.invalid_user"
msgstr "Одного из перечисленных пользователей не существует"

#: vk_bot/service/handlers.py:104
msgid "cmd.help"
msgstr "помощь"

#: vk_bot/service/handlers.py:108
msgid "cmd.help_message"
msgstr ""
"Список команд:\n"
"1. Должен\n"
"Формат: @пользователь1[,@пользователь2, …] должен [@пользователь3] сумма "
"[ежемесячно] название\n"
"2. Заплатить\n"
"Формат: [@пользователь1] заплатить"

#: vk_bot/service/commands.py:16
msgid "exception.owe_creation"
msgstr "Нельзя создавать долги в личных сообщениях"

#: vk_bot/service/temp.py:82
msgid "exception.unknown"
msgstr "Произошла неизвестная ошибка :("
